dashboard/dashboard.go

```diff
@@ -67,6 +71,14 @@ type Dashboard struct {
 
 	quit chan chan error // Channel used for graceful exit
 	wg   sync.WaitGroup  // Wait group used to close the data collector threads
+
+	peerCh  chan p2p.MeteredPeerEvent // Peer event channel.
+	subPeer event.Subscription        // Peer event subscription.
+
+	syncMode downloader.SyncMode
```

Comment:
Pls remove the sync mode, that's mostly an internal detail, is not that useful on the outside, at least not now.